website/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  // This file is not used in compilation. It is here just for a nice editor experience.
+  "extends": "@tsconfig/docusaurus/tsconfig.json",
```

Comment:
We should extend the `tsconfig.json` located in the root directory (`../tsconfig.json`).

Reply:
It would not work actually because Docusaurus doesn't use this file to build it:

> Docusaurus doesn't use this tsconfig.json to compile your project. It is added just for a nicer Editor experience, although you can choose to run tsc to type check your code for yourself or on CI.
https://docusaurus.io/docs/typescript-support#setup

Reply:
Oh, okay! Thanks for the explanations! :)